spython/main/parse/converters.py

```diff
@@ -209,6 +209,12 @@ def docker2singularity(self, runscript="/bin/bash", force=False):
 
     # Take preference for user, entrypoint, command, then default
     runscript = self._create_runscript(runscript, force)
+
+    # If a working directory was used, add it as a cd
+    if self.workdir != None:
```

Comment:
I suggest `if self.workdir`. Otherwise at least use `if self.workdir is not None` (see pylint)

Reply:
Also fixed.